### ISSUE TITLE
10 a11y warnings with svelte 4.0.0

### ISSUE THREAD

**Issue by Ennoriel** [Ennoriel/svelte-awesome-color-picker#43]
- [x] A11y: `<div>` with mousedown handler must have an ARIA role (Picker.svelte:104:1)
- [x] A11y: `<div>` with mousedown handler must have an ARIA role (Slider.svelte:84:1)
- [x] A11y: `<div>` with mousedown handler must have an ARIA role (Alpha.svelte:87:1)
- [x] A11y: Non-interactive element `<label>` should not be assigned mouse or keyboard event listeners. (Input.svelte:9:0)
- [x] A11y: noninteractive element cannot have nonnegative tabIndex value (A11yNotice.svelte:16:1)

From the issue below:

- [x] operation hints should not be in the aria-label (you have the hint for arrow key operation there).
- [x] the normal sliders do not need a hint for arrow key operation (you currently have it there)
- [x] the colorpicker should not be opened automatically when the buttons are focused, but only after activating the buttons - a keyboard user might want to navigate over the button and not be "trapped" in the colorpicker
- [x] the colorpicker should be able to be closed with ESC
- [ ] for blind users the exceeding of a limit value concerning AA or AAA WCAG contrast requirements can be transmitted e.g. via Live Region

**Comment by Ennoriel** [comment]
aria attributes for color pickers have also been discussed here: https://github.com/w3c/aria-practices/issues/2742
